UtilityApps/src/converter.cpp

```diff
@@ -98,6 +102,8 @@ namespace {
       run_plugin(description,"DD4hepGeometry2GDML",output,&argv[output]);
     else if ( compact2pand )
       run_plugin(description,"DD4hepGeometry2PANDORA",output,&argv[output]);
+    else if ( compact2tgeo )
+      run_plugin(description,"DD4hepGeometry2TGeo",argc-output,&argv[output]);
```

Comment:
I certainly don't understand the logic behind `argc-output` here, but also not really behind the `output` arguments in the other cases.

Reply:
Thank you for the explanation!
Let's leave things as they are then!